wpilibj/src/athena/cpp/lib/CANJNI.cpp

```diff
@@ -8,7 +8,9 @@
 #include <assert.h>
 #include <jni.h>
 
+#ifdef CONFIG_ATHENA
```

Comment:
We do not, because that is directly accessing the NetComm function, which isn't included at all in sim. We could add a shim to all platforms if we wanted to.